src/vmm/src/vmm_config/mod.rs

```diff
@@ -20,6 +20,8 @@ pub mod boot_source;
 pub mod drive;
 /// Wrapper for configuring the entropy device attached to the microVM.
 pub mod entropy;
+/// Wrapper over hotplug configuration.
+pub mod hotplug;
```

Comment:
I assume this needs to be `[cfg(target_arch = "x86_64")]`. Please bring it over to the first commit.